packs/feat-effects/stance-stretching-reach.json

```diff
@@ -0,0 +1,52 @@
+{
+    "_id": "IhwcqxPt1SkgXQB0",
+    "img": "icons/skills/melee/strike-flail-destructive-yellow.webp",
+    "name": "Stance: Stretching Reach",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Stretching Reach]</p>\n<p>While in this stance, when you wield a melee weapon that requires two hands and doesn't have reach, the weapon gains a reach of 10 feet.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "encounter",
+            "value": -1
+        },
+        "level": {
+            "value": 5
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Howl of the Wild"
+        },
+        "rules": [
+            {
+                "itemType": "weapon",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:equipped",
+                    "item:melee",
+                    "item:usage:hands:2",
+                    {
+                        "not": "item:trait:reach"
+                    }
+                ],
+                "property": "traits",
+                "value": "reach-10"
```

Comment:
Does this even work on PCs?

Reply:
It seems to when added via `ItemAlteration`. Not sure it's considered valid though? Should I switch to just `"reach"`?

Reply:
<img width="296" alt="Screen Shot 2024-09-12 at 9 10 02 AM" src="https://github.com/user-attachments/assets/f37b8553-9faa-4da9-bc41-edaaa82aa5cb">

Reply:
Yeah, use reach